shared/src/models/utils/role.ts

```diff
@@ -5,3 +5,9 @@ export const roleAllowedValues: AllowedValues<Role> = {
     participant: true,
     trainer: true,
 };
+export type SpecificRole = 'eoc' | 'map-operator' | 'trainer';
+export const specificRolesAllowedValues: AllowedValues<SpecificRole> = {
```

Comment:
Use consistent naming

```suggestion
export const specificRoleAllowedValues: AllowedValues<SpecificRole> = {
```